CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.3.6
+
+* Extend `run_test` to use `CountryTaxBenefiSystem` _or_ `country_tax_benefit_system` from country openfisca package
```

Comment:
See the motivation of the PR.
Sometimes your country_tax_benefit_system is not an instance of a CountryTaxBenefitSystem but a reform